test/SlowTests/Tests/Spatial/DynamicSpatialQueries.cs

```diff
@@ -0,0 +1,447 @@
+﻿using FastTests;
+using Raven.Client.Documents.Indexes.Spatial;
+using Raven.Client.Documents.Queries;
+using Raven.Server.Documents.Queries;
+using Sparrow.Json;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Tests.Spatial
+{
+    public class DynamicSpatialQueries : RavenTestBase
+    {
+        public DynamicSpatialQueries(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public void VerifySingleSpatialPropertyInResultsTest()
```

Comment:
do we need word test at the end? I think most test cases dones't have that.

Reply:
oh - I thought that this is a convention